Plugins/Images/Images/NCER.cs

```diff
@@ -357,6 +384,13 @@ void Update_Struct()
             ncer.cebk.section_size = 0x20 + size;
             if (ncer.cebk.section_size % 4 != 0)
                 ncer.cebk.section_size += (4 - (ncer.cebk.section_size % 4));
+            
+            // Update partition data info
+            ncer.cebk.partition_data_offset = ncer.cebk.section_size - 8;
+            ncer.cebk.max_partition_size = max_partition_size;
+            ncer.cebk.first_partition_data_offset = (ncer.cebk.first_partition_data_offset > 0) ? ncer.cebk.first_partition_data_offset : 8;

```

Comment:
Does this means that the `first_partition_data_offset` can't be smaller than 8? In that case, should be check if it's smaller than 8 instad of 0?